app/models/activity.rb

```diff
@@ -4,6 +4,8 @@ class Activity < ApplicationRecord
 
   belongs_to :trackable, polymorphic: true, required: false
 
+  def project=(new_project); end
```

Comment:
We may be using it in nodes, boards, activities in CE. Sounds like a possible model concern. Added a [NICE TO HAVE] task for it.